sdk/communication/azure-communication-sms/CHANGELOG.md

```diff
@@ -1,14 +1,14 @@
 # Release History
 
-## 1.2.0-beta.1 (Unreleased)
+## 1.1.2 (2022-05-11)
 
-### Features Added
-
-### Breaking Changes
+### Other Changes
 
-### Bugs Fixed
+#### Dependency updates
+- Upgraded `azure-communication-common` to 1.1.3
+- Upgraded `azure-core` to 1.28.0
+- Upgraded `azure-identity` to 1.5.1
```

Comment:
We don't need to add updates about our test dependencies.

Reply:
I don't understand. This did get updated in the pom.xml. I feel it's worth including this.